token/cli/src/main.rs

```diff
@@ -807,6 +852,50 @@ fn main() {
                 .nonce_args(true)
                 .offline_args(),
         )
+        .subcommand(
+            SubCommand::with_name("create-multisig")
+                .about("Create a new account describing an M:N multisignature")
+                .arg(
+                    Arg::with_name("minimum_signers")
+                        .value_name("MINIMUM_SIGNERS")
+                        .validator(is_multisig_minimum_signers)
+                        .takes_value(true)
+                        .index(1)
+                        .required(true)
+                        .help(&format!("The minimum number of signers required \
+                            to allow the operation. [{} <= M <= N]",
+                            MIN_SIGNERS,
+                        )),
+                )
+                .arg(
+                    Arg::with_name("signatory")
+                        .value_name("SIGNATORY_PUBKEY")
+                        .validator(is_valid_pubkey)
+                        .takes_value(true)
+                        .index(2)
+                        .required(true)
+                        .min_values(MIN_SIGNERS as u64)
+                        .max_values(MAX_SIGNERS as u64)
+                        .help(&format!("The public keys of all signatories allowed \
+                            to sign for this account. [{} <= N <= {}]",
+                            MIN_SIGNERS, MAX_SIGNERS,
+                        )),
+                )
+                .arg(
+                    Arg::with_name("address_keypair")
+                        .long("address-keypair")
+                        .value_name("ADDRESS_KEYPAIR")
+                        .validator(is_valid_signer)
+                        .takes_value(true)
+                        .help(
+                            "Specify the address keypair. \
+                             This may be a keypair file or the ASK keyword. \
+                             [default: randomly generated keypair]"
```

Comment:
Is your thinking that users explicitly pass `--owner <base58-address-string>` when they use `--multisig-signer` args? I'm just wondering if we should make it harder to create a multisig account without a specified address_keypair, so that cli use is easier and more consistent with single owners, in terms of setting config, etc.

Reply:
Yep!  `--owner` must be specified explicitly (there's a `.requires("owner")` on `--multisig-signer`), due to its playing either a signing or non-signing role, depending on whether or not its a multisig.  We don't have a way to support this in our signer resolution utils yet, so I went with [this hack](https://github.com/solana-labs/solana-program-library/pull/713/files#diff-d587635f3541d3409d5226615ee6e533c17209eb5192e6271b4a6e651e74f0c3R1337-R1349) for now, with the intent of finding a more general solution upstream.

I think the only ergonomic way to make it harder to use a throwaway keypair address would be to make it impossible (which I'm totally cool with!).  I don't believe we can resolve both optional and variable-count positional args on the same command line

Reply:
Ah I did overlook that `requires` on multisig-signer. I vote for no throwaway keypair for multisigs. I think we could potentially eliminate that `requires`... what do you think about checking that owner is a multisig account when `--multisig-signer` is provided?

Reply:
> I vote for no throwaway keypair for multisigs.

My only hesitation is that this makes multisig special.  It'd be nice to stay consistent across all of the create subcommands.  @mvines how sad would you be if the default throwaway addresses went away across the board?

> I think we could potentially eliminate that requires... what do you think about checking that owner is a multisig account when --multisig-signer is provided?

Yeah I guess we can just fallback to the CLI config keypair, even though it's probably not what anyone will ever want

Reply:
`requires("owner")` is gone